merlin-server/constraints-dsl-compiler/src/libs/mission-model-generated-code.ts

```diff
@@ -38,9 +38,9 @@ export class ActivityInstance<A extends ActivityType> {
     this.parameters = 5;
   }
 
-  public window(): Windows {
-    return new Windows({
-      kind: AST.NodeKind.WindowsExpressionActivityWindow,
+  public span(): Spans {
+    return new Spans({
+      kind: AST.NodeKind.SpansExpressionActivitySpan,
       alias: this.__alias
```

Comment:
The `ActivityInstance` placeholder code here can be completely removed, I must have forgotten to remove it when we did that rework of the types and made it so this class didn't need to be generated.

Reply:
but we use this `span()` in the `during()`.

Reply:
Oh. In the mission model.

Reply:
`ActivityInstance` is now in the fluent-api file. This is a duplicate class that isn't used.